restapi/stories.go

```diff
@@ -58,6 +58,42 @@ func (api api) ReturnStoryByID(w http.ResponseWriter, r *http.Request) render.Re
 	return rest.JSONStatusOK(story)
 }
 
+func (api api) DeleteStoryByID(w http.ResponseWriter, r *http.Request) render.Renderer {
+	// TODO: Delete the image from S3 properly
+	var story models.Story
+	id := chi.URLParam(r, "storyID")
+
+	err := api.database.Preload("Author").Where("id=?", id).First(&story).Error
```

Comment:
Since this can cause issues in tests maybe we should just query the author model instead?

Reply:
I think i could just remove preload author because the primary key (first name, last name, and origin country) is already in returned story

Reply:
nvm i lied. We have to cuz need it to for deleting author